client/app/pages/dashboards/components/DashboardHeader.jsx

```diff
@@ -193,8 +193,12 @@ function DashboardControl({ dashboardConfiguration, headerExtra }) {
           {showRefreshButton && <RefreshButton dashboardConfiguration={dashboardConfiguration} />}
           {showFullscreenButton && (
             <Tooltip className="hidden-xs" title="Enable/Disable Fullscreen display">
-              <Button type={buttonType(fullscreen)} className="icon-button m-l-5" onClick={toggleFullscreen}>
-                <i className="zmdi zmdi-fullscreen" />
+              <Button
+                type={buttonType(fullscreen)}
+                className="icon-button m-l-5"
+                onClick={toggleFullscreen}
+                aria-label="Fullscreen display">
```

Comment:
```suggestion
                aria-label="Toggle Fullscreen">
```
?

Reply:
It's actually different, because it doesn't cause the app to enter fullscreen, just to hide accessory elements "fullscreen mode"

Reply:
My concern is that "Fullscreen display" is not an action 😕